debian/kafka-connect-base/include/etc/confluent/docker/configure

```diff
@@ -54,6 +54,24 @@ then
   dub ensure CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL
 fi
 
+if [ -z "${CONNECT_SSL_KEYSTORE_FILENAME+x}" ]
+then
+  dub ensure CONNECT_SSL_KEYSTORE_CREDENTIALS
+  CONNECT_SSL_KEYSTORE_CREDENTIALS_LOCATION="/etc/kafka/secrets/$CONNECT_SSL_KEYSTORE_CREDENTIALS"
+  dub path "$CONNECT_SSL_KEYSTORE_CREDENTIALS_LOCATION" exists
+  export CONNECT_SSL_KEYSTORE_PASSWORD
+  CONNECT_SSL_KEYSTORE_PASSWORD=$(cat "$CONNECT_SSL_KEYSTORE_CREDENTIALS_LOCATION")
+fi
+
+if [ -z "${CONNECT_SSL_TRUSTSTORE_FILENAME+x}" ]
+then
+  dub ensure CONNECT_SSL_TRUSTSTORE_CREDENTIALS
+  CONNECT_SSL_TRUSTSTORE_CREDENTIALS_LOCATION="/etc/kafka/secrets/$CONNECT_SSL_TRUSTSTORE_CREDENTIALS"
+  dub path "$CONNECT_SSL_TRUSTSTORE_CREDENTIALS_LOCATION" exists
+  export CONNECT_SSL_TRUSTSTORE_PASSWORD
+  CONNECT_SSL_TRUSTSTORE_PASSWORD=$(cat "$CONNECT_SSL_TRUSTSTORE_CREDENTIALS_LOCATION")
+fi
+
```

Comment:
We should also add the sections for the truststore and keystore file just like Kafka for consistency. 
https://github.com/confluentinc/cp-docker-images/blob/3.2.x/debian/kafka/include/etc/confluent/docker/configure#L72-L74
https://github.com/confluentinc/cp-docker-images/blob/3.2.x/debian/kafka/include/etc/confluent/docker/configure#L88-L90 

This might break for existing users who are using `CONNECT_SSL_KEYSTORE_LOCATION` to pass the filepath directly so we might want to make it conditional on whether `CONNECT_SSL_TRUSTSTORE_FILENAME` is set or not. But the check `dub file 
 CONNECT_SSL_KEYSTORE_LOCATION exists` makes sense in any case.